### DISCUSSION TITLE
Naming the "Kitchensink" image

### DISCUSSION THREAD

**Discussion by PhillypHenning** [bitovi/bitops#207] [Polls]
Does anybody have a word or phrase to explain the english phrase;
“Everything but the kitchen sink”
Context:
We are working on naming a BitOps image that will contain all of our supported plugins. The idea behind this is that this should be an image that regular users can grab and use right away without much configuration involved.

**Comment by mickmcgrath13** [comment]
the following aren't appropriate, imo, because the pre-built images aren't necessarily going to have everything in them:
- Complete
- Exhaustive

The following is ambiguous, imo, because there are various "official" things that we'll provide (like the base image), and we might not include all "offical" plugins in the aggregate image:
- Official

**Comment by arm4b** [comment]
Yeah, it makes really sense to rename it to something that definitely feels right to everyone, otherwise, we'll just get more confusion. 

I usually avoid codenames as the focus is on a wider community and different cultures, languages or perceptions might be different internationally. And so that's why the naming question.

Also voted for "kitchensink" for now to avoid breaking things and renaming it with no good alternatives.

**Comment by mickmcgrath13** [comment]
Locking this discussion.  Resolution: Omnibus

**Reply by mickmcgrath13** [reply]
oh, it was already locked. thanks @PhillypHenning
